README.md

```diff
@@ -426,6 +426,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**PilferShush Jammer**](https://github.com/kaputnikGo/PilferShushJammer) <sup>**[[F-Droid](https://www.f-droid.org/app/cityfreqs.com.pilfershushjammer)]**</sup>
 * [**Privacy Indicator**](https://github.com/NitishGadangi/Privacy-Indicator-App) <sup>**[[F-Droid](https://f-droid.org/app/com.nitish.privacyindicator)]**</sup>
 * [**RustDesk**](https://github.com/rustdesk/rustdesk) <sup>**[[F-Droid](https://f-droid.org/app/com.carriez.flutter_hbb)]**</sup>
+* [**Record You**](https://github.com/Bnyro/RecordYou)
```

Comment:
[No F-Droid](https://github.com/Bnyro/RecordYou/issues/39) unfortunately :-(

Reply:
Ah, that is why i only added github link.
He says tho, it is as same as in izzydroid.

Are you sure you want me to remove Record you?

I'm fine with it, no issues here.

Reply:
Sorry, no! :see_no_evil: 

The message was intended as info only, so nothing to do here :-).